README.md

```diff
@@ -87,6 +87,23 @@ Show message with ID 1
 
     RETR 1
 
+## Use with docker-machine
+
+Because service clients, like the addon-administration, that are using the
+3-legged flow to authenticate a user with a redirect to OSIAM only know one
+endpoint to redirect to and back to themselves. This container will only
```

Comment:
The sentence doesn't really make sense. either replace the full stop with a comma or reformulate. I have trouble understanding it.